tests/unit/ember-sinon-qunit-test.js

```diff
@@ -56,3 +56,15 @@ test('sinon sandbox cleans up after itself', function (assert) {
   assert.ok(!spy.called, 'spy has no registered calls');
   assert.ok(!stub.called, 'stub has no registered calls');
 });
+
+test('does not fake timers by default', function (assert) {
+  assert.notEqual(Date.now(), 0, 'Date.now() is not reset');
+});
+
+test('does fake timers on request', { useFakeTimers: true }, function (assert) {
```

Comment:
I'm not sure how I feel about passing an options hash to each individual test. If you're using fake timers a lot, that could get onerous. Could you see if it's possible to simply import `sinon` and change its config in `beforeEach` of the test module?